### PR TITLE
Modified query for component analysis

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

File: bayesian/utils.py [hunk 1]
```diff
@@ -327,18 +327,19 @@ def search_packages_from_graph(tokens):
...
-.by(valueMap()),select('package','version').by(valueMap())).fill(results);
-[epv:results,recommended_versions:no_cve_versions];\
+    script1 = """\
+g.V().has('pecosystem', ecosystem).has('pname', name).has('version', version).as('version')\
+.in('has_version').dedup().as('package').select('version').coalesce(out('has_cve')\
+.as('cve').select('package','version','cve').by(valueMap()),select('package','version')\
+.by(valueMap()));\
 """
 
+    script2 = """\
```
Comment (miteshvp) [inline bayesian/utils.py]: Get the payload and script in the if loop

File: bayesian/utils.py [hunk 2]
```diff
@@ -347,7 +348,44 @@ def get_analyses_from_graph(ecosystem, package, version):
...
+
+            if not (resp['result'].get('data') and len(resp['result'].get('data')) > 0):
+                # trigger unknown component flow in API for missing package
+                return None
+
+            clubbed_data.append({
+                "epv": resp['result']['data']
+            })
+            second_query_flag = False
+            for first_data in resp['result'].get('data'):
```
Comment (miteshvp) [inline bayesian/utils.py]: you can directly use if 'cve' in resp['result'].get('data')[0]

**Comment by tisnik** [comment]
I'm sorry, but the new code is not covered by unit test.

**Comment by miteshvp** [comment]
@tisnik - good catch. 
@yzainee - Please add the relevant unit test case here

**Pull request merged**